railties/lib/minitest/rails_plugin.rb

```diff
@@ -52,6 +52,11 @@ def self.plugin_rails_init(options)
     if reporter.reporters.reject! { |reporter| reporter.kind_of?(ProgressReporter) }
       reporter << ::Rails::TestUnitReporter.new(options[:io], options)
     end
+
+    if ENV["CI"]
+      require "minitest/reporters"
```

Comment:
I am neutral to this proposal itself. 
But this affects apps that don't use existing `minitest-reporters`. Please do not load libraries that are not dependent.

Reply:
True. I'll fix.